.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pyc
+*.~*
```

Comment:
I'm curious what files this is for?

Reply:
Emacs backup files, which are foo.bar.~N~, and also `vi` backup files, which I recall being `foo.bar.~`.